lib/SP/Providers/Auth/Ldap/LdapActions.php

```diff
@@ -159,16 +159,19 @@ protected function getGroupFromParams(): string
      *
      * @return bool|array
      */
-    protected function getResults($filter, array $attributes = null)
+    protected function getResults($filter, array $attributes = null, $searchBase = false)
     {
         $cookie = '';
         $results = [];
 
         do {
             ldap_control_paged_result($this->ldapHandler, 1000, false, $cookie);
 
-            $searchRes = @ldap_search($this->ldapHandler, $this->ldapParams->getSearchBase(), $filter, $attributes);
-
+            if ($searchBase == false) {
+                $searchBase = $this->ldapParams->getSearchBase();
+            }
+            $searchRes = @ldap_search($this->ldapHandler, $searchBase, $filter, $attributes);
```

Comment:
ldap_search function expects an string on third parameter. Default value is set to `boolean` when it isn't set

Reply:
It should be safe to set it to `null`